packages/interpreter/src/helpers/index.ts

```diff
@@ -62,6 +62,10 @@ export function getOperationValue(
 
     case "/=":
     case "/":
+      if (operands.right === 0) {
+        throw new RuntimeException(`Bhai Bhai Bhai Bhai Bhai ye kya kar diya...`)
```

Comment:
can we change this error message to => `Bhai Bhai Bhai ye kya kar diya...zero se divide ni karte`?

5 bhai's seems, little too much. Also we should tell the developer what is he doing wrong (zero division).

Reply:
Sure that makes sense!